global.json

```diff
@@ -1,11 +1,11 @@
 {
   "sdk": {
-    "version": "9.0.100",
+    "version": "9.0.102",
```

Comment:
FYI - @joperezr - I updated the SDK and runtime previous versions for testing to the latest patches as well in this PR.